### DISCUSSION TITLE
A searchbar like in bookmarks sidebar? possibly with regex?

### DISCUSSION THREAD

**Discussion by lepczynski** [piroor/treestyletab#3628] [Ideas]
Hi I was wondering, what do yall think of such feature?
I tend to have >10 windows open, 1 of them with >100tabs so sometimes it would be great to see weather I already have a research open on a given topic..
I am a developer, only for Java backends, but I have meddled with JS before :> I would love to help if I had sb willing to mentor me in this (never coded FFox plugins)

**Comment by irvinm** [comment]
There was 1 addon already done as a sub-panel to TST that did exactly that, but that functionality was broken in TST 4.0 with how the tabs are rendered.  (https://addons.mozilla.org/firefox/addon/tst-search/)

Honestly, the best solution in the short-term is probably to do searches via the "awesome bar" (URL bar) with the "%" delimiter to search only tabs.  

However, there are a handful of addons for general tab searching (not TST related) that also do a nice job like:  https://addons.mozilla.org/en-US/firefox/addon/tabby-window-tab-manager (and it looks to support [fancy-regex](https://github.com/fancy-regex/fancy-regex))

**Reply by piroor** [reply]
FYI, I've opened PR for TST Tab Search: https://github.com/NiklasGollenstede/tst-search/pull/38
